assistanceTools/zendesk.json

```diff
@@ -92,6 +92,20 @@
           "it-IT": "Privacy",
           "en-EN": "Privacy"
         }
+      },
+      {
+        "value": "suggerimento",
+        "description": {
+          "it-IT": "Suggerimento",
+          "en-EN": "Suggestion"
+        }
+      },
+      {
+        "value": "nessuno_dei_precedenti",
```

Comment:
🤓 **suggestion**:
`nessuno_dei_precedenti` implies that the reader knows what the preceding categories are. I suggest using something more generic as could be `generic` itself or `uncategorized`.

Reply:
@pietro909 These categories are reported by app IO in the same order as the JSON array.
Also, we have to implement this requirement: (see last two points)
https://pagopa.atlassian.net/browse/IA-667
Thanks, Marco

Reply:
Looking at the task you mentioned, I see only 2 points out of 4 addressed in this PR. 
Is this what we want?

Reply:
@pietro909 Correct. The remaining points (Accorpare...) will be addressed later.